conf/battle/player.conf

```diff
@@ -60,10 +60,17 @@ natural_healsp_interval: 8000
 // Automatic healing skill's time interval. (in milliseconds)
 natural_heal_skill_interval: 10000
 
-// The maximum weight for a character to carry when the character stops healing naturally. (in %)
+// The maximum weight for a character to carry before the it stops healing naturally. (in %)
 // For renewal: Requires client 20171025 or newer to display properly
```

Comment:
Probably should instead say that a client of 20171025 or newer displays "70%" and an older client displays "50% on the icon regardless of this setting. (At least I'm assuming this is what this hint here means.)

Reply:
Actually it depends on the ¹«°Ô50ÀÌ»ó.tga icon. Gravity changed the number in the icon, but not the name file.
I'll try to explain it there.